chainerx_cc/chainerx/routines/math.cc

```diff
@@ -544,6 +575,10 @@ Array Minimum(const Array& x1, Scalar x2) {
 
 Array Minimum(Scalar x1, const Array& x2) { return Minimum(x2, x1); }
 
+Array Minimum(const Array& x1, const Array& x2) {
+    return IfGreaterElse(x1, x2, x2, x1);  // x1 > x2 ? x2 : x1
```

Comment:
Sure! Will work on it @asi1024 .

Reply:
@asi1024 Do I define `BroadcastQuaternary` to handle 4 inputs in case of `Maximum` and `Minimum` ?

> It fails on the tests for different shapes. Could you broadcast the inputs with `BroadcastBinary`?